src/test/test-condition.c

```diff
@@ -37,66 +37,82 @@ static void test_condition_test_path(void) {
         Condition *condition;
 
         condition = condition_new(CONDITION_PATH_EXISTS, "/bin/sh", false, false);
+        assert_se(condition);
```

Comment:
true, but it trips me up when i look at the code... I mean, we generally check OOM everywhere, on all constructors. We really should do that here too. If it's not me who's the next one tripping up on this I am sure somebody welse will, sooner or later.